### DISCUSSION TITLE
What do you mean by 'Soho Vibes'?

### DISCUSSION THREAD

**Discussion by prmadev** [rose-pine/rose-pine-theme#71] [Q&A] [answered]
Hey! 
Thank for the elegant, delightful and beautiful themes and color schemes! 
Sorry if my question is related to something obvious, but I have never heard of this term and my searches were unfruitful. 
Would you mind explaining this term?

**Comment by mvllow** [comment, marked as answer]
"Cozy-yet-luxurious style". In other words, a more functional and natural take on modern/minimal design.

![image](https://user-images.githubusercontent.com/1474821/159933232-cc4d9e62-987d-4a76-acef-6da2a0c42e63.jpeg)

**Reply by prmadev** [reply]
Ah! I see! That makes perfect sense to me now! I definitely can get on-board.
